server/src/main/java/org/hyperskill/community/flashcards/card/model/Card.java

```diff
@@ -0,0 +1,39 @@
+package org.hyperskill.community.flashcards.card.model;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.time.Instant;
+
+/**
+ * Basic class for all flashcards, contains fields shared among all subclasses
+ */
+@Getter
+@Setter
+@Document
+@ToString
+public abstract class Card {
+    @Id
+    protected String id;
+    protected String author;
+    protected boolean isPublic;
+    protected String title;
+    protected String category;
+    protected String question;
+    @CreatedDate
```

Comment:
do we need to store the category - or isn't this more the collection, the document is attahced to - i.e. do we need this back-reference?

Reply:
Agree, removed this backing field,